exercises/hello-world/TUTORIAL.md

```diff
@@ -503,39 +282,11 @@ Here's an up-to-date list of submissions on the Java track:
 
 http://exercism.io/tracks/java/exercises
 
-
-
-## Submit another iteration
-
-You are also encouraged to consider additional "requirements" on a given
-exercise.
-
-For example, you could add a test or two that requires that the greeting
-use the capitalized form on the person's name, regardless of the case they
-used.
-
-In that situation, you'd write a test to set-up that new expectation and
-then implement that in the code (the same process we just went through
-together, above).
-
-
-
 ## Contribute to Exercism
 
-The entire of Exercism is Open Source and is the labor of love for over
+The entire of Exercism is Open Source and is a labor of love for over
 100 maintainers and many more contributors.
 
 A starting point to jumping in can be found here:
 
 https://github.com/exercism/problem-specifications/blob/master/CONTRIBUTING.md
```

Comment:
This is a different link to the one I've used in the general read me: https://github.com/exercism/docs/tree/master/contributing-to-language-tracks. We should probably be consistent and stick to one of them :P Yours looks easier to read since it's one file instead of many. But the [porting an exercise](https://github.com/exercism/problem-specifications/blob/master/CONTRIBUTING.md#porting-an-exercise-to-another-language-track) link is broken, and that's one of the most useful bits of the documentation in my opinion.